exercises/robot-name/HINTS.md

```diff
@@ -0,0 +1,17 @@
+This exercise introduces work with Julia's [type system](http://docs.julialang.org/en/stable/manual/types/)
+and random [numbers](http://docs.julialang.org/en/stable/stdlib/numbers/).
+
+Let's create a structure that will describe our robots
+and then use it to create new instances and to work with them.
+
+Generate the robot's name using randomness and don't worry about
+collisions, tests sometimes will fail and that's alright.
```

Comment:
Perhaps add a comment that one should not worry about collisions at first but then try to resolve it later?